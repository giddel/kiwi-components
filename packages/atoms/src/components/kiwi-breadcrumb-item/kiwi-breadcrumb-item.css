kiwi-breadcrumb-item:not(:last-child) li::after {
  content: 'Â»';
  padding: 0 0.75rem;
}
