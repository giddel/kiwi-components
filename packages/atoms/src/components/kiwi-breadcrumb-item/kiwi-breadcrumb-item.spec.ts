import { newSpecPage } from '@stencil/core/testing';
import { KiwiBreadcrumbItem } from './kiwi-breadcrumb-item';

describe('kiwi-breadcrumb-item', () => {
  it('renders', async () => {
    const page = await newSpecPage({
      components: [KiwiBreadcrumbItem],
      html: `<kiwi-breadcrumb-item></kiwi-breadcrumb-item>`,
    });
    expect(page.root).toMatchSnapshot();
  });
});
