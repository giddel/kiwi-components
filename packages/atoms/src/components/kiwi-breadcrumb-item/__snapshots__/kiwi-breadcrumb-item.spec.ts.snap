// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`kiwi-breadcrumb-item renders 1`] = `
<kiwi-breadcrumb-item>
  <!---->
  <li class="breadcrumb-item"></li>
</kiwi-breadcrumb-item>
`;
